Article formatting guidelines (#6668)
fleetdm/fleet: handbook/brand.md

2. Root Cause: Why did this error happen?
3. Side effects: did this error have any side effects? e.g., did it corrupt any data? Did code that was supposed to run afterward and “finish something up” not run, and did it leave anything in the database or other systems in a broken state requiring repair? This typically involves checking the line in the source code that threw the error.

# Article formatting guidelines

> To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in <ins><strong>Markdown</strong></ins>.

Author: I'm assuming that "<ins><strong>Markdown</strong></ins>" is destined to be a link.

Reviewer: Yes, this will be updated to link to your Markdown guide. Please add "(TODO)" after it for now.

> To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in <ins><strong>Markdown</strong></ins>.

Author: ```suggestion
To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in [Markdown](TODO).
```

### Hero image
Consider adding a hero image for a more significant impact. Get in touch with Digital Experience via #content on Slack to make a request. 

### Table of contents
For long articles or guides, consider adding a table of contents.

### Introduction
It’s good practice to start your article with a clear summary of what you will be discussing.

### Main content
The main body of your article.

### Conclusion
It’s a good idea to finish your article with a clear closing statement.

### Add a customizable CTA
Add a CTA at the end of your article. See [Customizable CTA](#heading=h.2nnazmktmx29) below for instructions on creating a CTA tailored to your article topic.

## Images & screenshots
Images are a great way to help engage your readers. But consider the following before including images or screenshots in your article:

- Does the image add value?
- Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.

## Meta tags

> These tags help … summary of what these are for (todo)

Author: Should "…" be filled in with something else?

Reviewer: Yes, please, can you grab a moment with @eashaw and ask him for a summary of what the Meta tags do?

Author: This is on the schedule.

- __Guides__: help articles for using and deploying Fleet.
- __Podcasts__: podcast-related posts.
- __Product__: posts related to Fleet features.
- __Releases__: release posts, security, and patch releases.
- __Reports__: posts about the industry, data, surveys, etc.
- __Security__: posts about how we approach security at Fleet and other security-related topics.
- Success stories: stories from users or customers successfully using Fleet. 

## `publishedOn`
The date that the article was published. Please follow the correct date format, e.g., __2021-09-29__.

## `articleImageUrl`
The relative url path for the article cover image. Article images are stored in ../website/assets/images/articles/ See <ins>fleetdm.com image conventions</ins> (todo).

## Customizable CTA
Use the following code snippet to include an inline CTA (call to action) in your article:

`<call-to-action 
title=”Give Fleet a try”
text=”Get up and running with a text environment of Fleet within minutes.”
primary-button-text=”Get started” 
primary-button-href=”/get-started?try-it-now” 
secondary-button-text=”Schedule a demo”
secondary-button-href=”calendly.com/fleetdm/demo”>
</call-to-action>`

> ___________INSERT_IMAGE_______________

Author: Having not put an image in Markdown before I could use a little guidance.

Reviewer: Ok, let's get together.

Every week, we run `npm audit --only=prod` to check for vulnerabilities on the production dependencies of fleetdm.com. Once we have a solution to configure GitHub's Dependabot to ignore devDependencies, this manual process can be replaced with Dependabot.

## Article formatting guidelines

> To publish an article, you will need to create a Pull Request for fleetdm.com with a file of your article, formatted in Markdown (todo).

Reviewer: ```suggestion
To publish an article, you will need to create a Pull Request for a new file, formatted in Markdown (todo), in https://github.com/fleetdm/fleet/tree/main/articles.
```

#### Hero image
Consider adding a hero image for a more significant impact. Get in touch with Digital Experience via #content on Slack to make a request. 

#### Table of contents
For long articles or guides, consider adding a table of contents.

#### Introduction
It’s good practice to start your article with a clear summary of what you will be discussing.

#### Main content
The main body of your article.

#### Conclusion
It’s a good idea to finish your article with a clear closing statement.

#### Add a customizable CTA
Add a CTA at the end of your article. See [Customizable CTA](#customizable-cta) below for instructions on creating a CTA tailored to your article topic.

### Images and screenshots
Images are a great way to help engage your readers. But consider the following before including images or screenshots in your article:

- Does the image add value?
- Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.

### Meta tags

> These tags help … summary of what these are for (todo)

Author: ```suggestion
These tags help pass information to the website about the article to display as well as store it. 
```

Reviewer: @Desmi-Dizney. You hit **Resolve conversation** again instead of **Commit suggestion** 🧐

Author: 🤦🏻‍♂️

> - Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.

Reviewer: ```suggestion
- Is your image likely to go out of date soon? (Consider the long-term maintenance of your article.)
```
